### ISSUE TITLE
Installed AVEncoder on VPS, Stuck At Install Page, Need Help

### ISSUE THREAD

**Issue by bbyrdhouse** [WWBN/AVideo-Encoder#305]
I have installed an encoder on a VPS that is separate from my live streamer. A completely separate host.
The install seemed to go ok even though the VPS uses Centos 7.9 ... Where I am stuck is the install page where it asks for the database connection credentials.

I want to know, what credentials is it looking for?
Is it looking for a new database on the VPS where the newly installed encoder is?
Or is it looking for the credentials of where the current livestreamer are located?

Any guidance would be appreciated, this is the first time I am doing this and if I can get it sorted I will write up a tutorial so that others can benefit.

Note the images below:
<img width="1234" alt="encode_1" src="https://user-images.githubusercontent.com/7376298/101174412-788c5380-3609-11eb-8119-b3b959351ace.png">
<img width="1277" alt="encode_2" src="https://user-images.githubusercontent.com/7376298/101174440-8215bb80-3609-11eb-83f2-609de0a0cbe4.png">

**Comment by DanielnetoDotCom** [comment]
Usually, the database credentials are provided by your host company or thru Cpanel.

I am not used to installing it thru any control panel, but I guess you can create a user somewhere

**Comment by bbyrdhouse** [comment]
@DanielnetoDotCom - I think you are misunderstanding my question ... I am not asking about how to set up a database ... I do that all the time.
I am asking about what is the image asking from me?

Is it asking me to create a database on the new encoder server or is it asking me for credentials to the database that you created when you set up my streaming server?

The instructions are not clear what credentials it wants

**Comment by bbyrdhouse** [comment]
I went ahead and made the assumption that it wanted me to create a database on the encoder server ... so that is what I did...  I couldn't get anyone to verify that this is what should be done. Evidently, that is what I was supposed to do because after I did, it finished installing.

Having to guess at what the instructions want you to do it time consuming and frustrating, perhaps the instructions should be changed to be more clear.
